geonetwork/stack.yml

```diff
@@ -12,9 +12,9 @@ services:
       image: geonetwork
       restart: always
       ports:
-          - 8080:8080
+         - 8080:8080
       environment:
-          DATA_DIR: /var/lib/geonetwork_data
+         - DATA_DIR=/var/lib/geonetwork_data
```

Comment:
@metazool  Could you change this to use keep the ":", instead of the "="? I agree with the other changes